news/remove-userconfig.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* `user_config.py` function.
```

Comment:
Since this is written for changelog.rst, both used by savvy users and future maintainers (could be new members) maybe a bit more context would be better as to what was changed (user config.py) which is stated and also maybe why this was  removed?